forms-flow-web/src/services/PdfService.js

```diff
@@ -22,6 +22,13 @@ export const exportToPdf = ({formId, formName, pdfName}) => {
 };
 
 export const printToPDF = ({ pdfName, formName }) => {
+  let maxWidth = '1440px';
+  if (window.location.href.includes('/task/')) {
+    maxWidth = '1920px';
+  }
+  const appContainer = document.querySelectorAll("div.app-container.container")[0];
+  appContainer.style.maxWidth = maxWidth;
```

Comment:
Just to be sure it doesn't break anything :)
```suggestion
  appContainer?.style?.maxWidth = maxWidth;
```

Reply:
This is for inline style, So that will be available both style and all its properties. But I guess I can check array length before processing that help if we add new page without app-container classes.